Tasks/GradleV2/task.json

```diff
@@ -16,7 +16,7 @@
     "author": "Microsoft Corporation",
```

Comment:
Need to update version of package `azure-pipelines-tasks-codecoverage-tools` to new one once PR https://github.com/microsoft/azure-pipelines-tasks-common-packages/pull/146 is merged and the package is published